cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	// future enabled flags
+	ServiceCatalog      = createFlag("service-catalog", SetBool, nil, nil, true)
```

Comment:
Could we not merge a pull request for #1202 first and merge this pull request without this option. Then we all can try the service-catalog via the extra flag option and see how stable it is. Then we can make a decision on whether we add --service-catalog as proper option or experimental one. 

After all there are two issues for this as well.